app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
Toán tử này  ||=  là như thế nào em

Reply:
Em trả lời qua ví dụ nha anh
a ||= b có nghĩa là nếu a mà khác nil thì kết quả là a còn nếu a nó là nil thì nó sẽ lấy kết quả là b